setup.py

```diff
@@ -34,7 +34,7 @@
         'bcrypt',
         'commonmark',
 
-        'Flask-Login==0.4.1',
+        'Flask-Login',
```

Comment:
Correct to remove the existing pin, but I think now we work only with `Flask-Login>0.4.1`.